scipy/_lib/_util.py

```diff
@@ -367,6 +366,7 @@ def __init__(self, pool=1):
             self.pool = pool
             self._mapfunc = self.pool
         else:
+            from multiprocessing import Pool
```

Comment:
Can you add a comment linking to this page and optionally a short description for the rationale, like:
```suggestion
            # importing multiprocessing at the top level can be slow,
            # see https://github.com/scipy/scipy/pull/12872
            # and https://bugs.python.org/issue32596
            from multiprocessing import Pool
```